manager/controllers/app/plotter_generator.go

```diff
@@ -47,44 +46,21 @@ type PlotterGenerator struct {
 	ProvisionedStorage map[string]NewAssetInfo
 }
 
-// AllocateStorage creates a Dataset for bucket allocation
+// AllocateStorage creates a new connection based on the selected account
 func (p *PlotterGenerator) AllocateStorage(item *datapath.DataInfo, destinationInterface *taxonomy.Interface,
 	account *fapp.FybrikStorageAccountSpec) (*fapp.DataStore, error) {
 	// provisioned storage
-	var genBucketName, genObjectKeyName string
+	var genName string
```

Comment:
pls add a comment that it's a `datasetName`.
BTW, it can be confused, in the `CreateConnection` method signature, this argument is called `datasetName`, but it is stored into `object_key`. Maybe better to call it `object_key`?

Reply:
changed to genName - it is a generated name representing a copied dataset, not necessarily an object key.